src/components/resource-selector/shared/files-grid-viewer.js

```diff
@@ -56,10 +65,17 @@ function FilesGridViewer({
             <ActionButton
               title={t('common:preview')}
               icon={<PreviewIcon />}
-              intent={ACTION_BUTTON_INTENT.success}
```

Comment:
We could do the same thing in `wikimedia-files-viewer.js` (the last green preview button left).

Reply:
Oh, thanks for spotting this, removed!